src/clientsettingsdlg.cpp

```diff
@@ -440,11 +440,11 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
 
     // MeterStyle combo box
     cbxMeterStyle->clear();
-    cbxMeterStyle->addItem ( tr ( "LEDs" ) );       // MT_LED
-    cbxMeterStyle->addItem ( tr ( "Bar" ) );        // MT_BAR
-    cbxMeterStyle->addItem ( tr ( "Narrow Bar" ) ); // MT_SLIM_BAR
-    cbxMeterStyle->addItem ( tr ( "Round LEDs" ) ); // MT_SLIM_LED
-    cbxMeterStyle->addItem ( tr ( "Small LEDs" ) ); // MT_SMALL_LED
+    cbxMeterStyle->addItem ( tr ( "Bar (Narrow)" ) );  // MT_SLIM_BAR
+    cbxMeterStyle->addItem ( tr ( "Bar (Wide)" ) );    // MT_BAR
+    cbxMeterStyle->addItem ( tr ( "LEDs (Stripe)" ) ); // MT_LED
+    cbxMeterStyle->addItem ( tr ( "LEDs (Small)" ) );  // MT_SMALL_LED
+    cbxMeterStyle->addItem ( tr ( "LEDs (Big)" ) );    // MT_SLIM_LED
```

Comment:
- The distinction still sounds a bit strange to me. Example: "You you like your LEDs rather as Stripes, Small or Big?" The words in brackets do not describe the same type of thing. I've got no better idea though. The following would be more accurate, but probably to long:
  - LEDs (striped)
  - LEDs (round, small)
  - LEDs (round, big)
  This may quickly get into the bike-shedding area, so we should do the best we can but should merge nevertheless even if we think it's not perfect... :)

- Should the adjectives be lowercase? (No idea, not a native speaker)

Reply:
> You you like your LEDs rather as Stripes, Small or Big?

What is "rather" doing in that?  And why make "Stripe" plural?  You're changing things and then complaining it doesn't sound consistent... ;)

Reply:
Happy to update this, and would like to update the enum entries at the same time with similar logic.
So should we take this proposal (with lowercase) as the option to go for?

(I am not a native speaker as well)

Reply:
> What is "rather" doing in that? And why make "Stripe" plural? You're changing things and then complaining it doesn't sound consistent... ;)

I was trying to place the different options in an English sentence to see how they would sound.
What I was trying to say:
- *Stripe* is not a size. There could be small or big stripes (in theory, I'm not suggesting we add them)
- *Small* and *Big* are sizes, but don't imply any form (in contrast to *Stripe*)

Reply:
I don't we have to worry about the total length of the text, as the "language" box just below the setting in the GUI will be wider. Good point that Small and Big no longer indicate the shape, so adding "Round" would make sense to me.